kahuna/public/js/crop/controller.js

```diff
@@ -66,7 +67,16 @@ crop.controller('ImageCropCtrl',
         // else undefined is fine
     };
 
-    ctrl.crop = function() {
+     ctrl.callCrop = function() {
+         //prevents return keypress on the crop button posting crop twice
+         if (!ctrl.cropping) {
+             crop();
+         } else {
+             return;
+         }
+     };
+
+    const crop = () => {
```

Comment:
`function` would make more sense, as if I call `callCrop` before this line, it'll fail as `crop` wouldn't have been defined yet.